spec/factories/events.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :event do
+    title { Faker::Lorem.sentence }
+    starts_at Time.now + 2.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.